### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by johnzheng1975**
Change the helloworld service/ deploy to real content. It can run well in my testing.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by frankbu** [comment]
The last line of this blog post points to the sample code used in the article: https://github.com/istio/istio/tree/release-1.7/samples/helloworld

Anyway, we generally do not want to update old blog posts. Blogs are considered single-point-in-time articles, and rarely updated. If we think this is a valuable enough doc, we could consider moving it (or adding something similar) to the [traffic managment tasks](https://istio.io/latest/docs/tasks/traffic-management/).

File: content/en/blog/2017/0.1-canary/index.md [hunk 1]
```diff
@@ -59,33 +60,51 @@ apiVersion: apps/v1
...
   template:
     metadata:
       labels:
         app: helloworld
         version: v1
     spec:
       containers:
-      - image: helloworld-v1
-        ...
+      - image: nginx
```
Comment (frankbu) [inline content/en/blog/2017/0.1-canary/index.md:79]: this is supposed to be the helloworld image at dockerhub.

**Comment by frankbu** [comment]
Closing because we do not want to update the blog.

@johnzheng1975 Please open an issue if you think it would be worthwhile to add something similar in the mainline docs (instead of just in a blog).

**Comment by johnzheng1975** [comment]
That's make sense - Do not change blog. Just leave this closed.
Thanks for your time.

**Pull request closed without merging**